server/hap/HapProcessCeilometer.cc

```diff
@@ -397,7 +397,7 @@ HatoholError HapProcessCeilometer::parseInstanceElement(
 	return HTERR_OK;
 }
 
-HatoholError HapProcessCeilometer::getAlarmList(void)
+HatoholError HapProcessCeilometer::getAlarmInfoTable(VariableItemTablePtr &trigTablePtr)
```

Comment:
thank you for comment.
I want to getAlarmTable.
collectAlarmList whether better. . .